### PR TITLE
cli: clarify error message when `jj untrack` argument is not ignored

### PR DESCRIPTION
**Pull request by martinvonz**
See commit

### CODE REVIEW

File: src/commands.rs
Comment (arxanas) [inline src/commands.rs:1817]: Why does `path` have to be cloned here?

**Review by arxanas — APPROVED** [review]

File: src/commands.rs
Comment (martinvonz) [reply src/commands.rs]: Good point, it doesn't need to be.  Fixed.

**Comment by martinvonz** [comment]
Now with e2e tests and a bugfix.

Silly GitHub doesn't seem to let me say that this branch is based on the "e2e" branch without also thinking that I want to merge into "e2e". I just want "clarify_untrack_message" to have to wait for "e2e" before it can be merged.

**Comment by martinvonz** [comment]
> Silly GitHub doesn't seem to let me say that this branch is based on the "e2e" branch without also thinking that I want to merge into "e2e". I just want "clarify_untrack_message" to have to wait for "e2e" before it can be merged.

I've worked around it by merging the base branch.

**Pull request merged**
